js/src/components/SettingsReplacementVariableEditors.js

```diff
@@ -60,9 +83,10 @@ class SettingsReplacementVariableEditors extends React.Component {
 	 *
 	 * Renders a settings replacement variable field in every given element in
 	 * the singleFieldElements props. It requires every element to have a data-
-	 * react-replacevar-field-id attribute and data-react-replacevar-page-type
-	 * to function properly. This attribute should point to and existing
-	 * (hidden) input in the DOM.
+	 * react-replacevar-field-id, data-react-replacevar-page-type-recommended
+	 * and data-react-replacevar-page-type-specific attributes to function
+	 * properly. This attribute should point to and existing (hidden) input in
+	 * the DOM.
```

Comment:
Not clear _which_ attributes should point to hidden fields. Also, "This" or "These"? "point to and existing" should be "point to an existing". I guess it's only about `data-react-replacevar-field-id`.